tests/unit/test_tracing.c

```diff
@@ -3,6 +3,8 @@
 #include "sentry_tracing.h"
 #include "sentry_uuid.h"
 
+#define IS_NULL(Src, Field)                                                    \
```

Comment:
maybe these helpers make sense in `sentry_testsupport.h`?

Reply:
this is too generic name.  I'd rather have a `sentry_key_is_null()` helper function which would be a lot more readable to me.  (have i mentioned i'm not a fan of macros?)

Reply:
seeing as there's already another helper macro in the tests right below, can i defer any relocations or changes to the macros to https://github.com/getsentry/sentry-native/pull/637?